shredstream.proto

```diff
@@ -8,9 +8,13 @@ message Heartbeat {
   // don't trust IP:PORT from tcp header since it can be tampered over the wire
   // `socket.ip` must match incoming packet's ip. this prevents spamming an unwitting destination
   shared.Socket socket = 1;
+
+  // regions for shredstream proxy to receive shreds from
+  repeated shared.Region regions = 2;
```

Comment:
i think string might be nice even with higher payload size, will save us from needing to update the region on every new block engine